nvm.sh

```diff
@@ -1926,6 +1926,9 @@ nvm_install_binary() {
     return 3
   fi
 
+  local nosource
+  nosource = "${4-}"
```

Comment:
```suggestion
  nosource="${4-}"
```
assignments won't work if there's spaces around the =